### DISCUSSION TITLE
Report when using gestures to mute or unmute the speech coming from the remote computer""

### DISCUSSION THREAD

**Discussion by cary-rowen** [NVDARemote/NVDARemote#258] [General]
Hello everyone,
We sometimes need to use gestures to mute or unmute the speech coming from the remote computer,
However, the current status cannot be reported after the gesture is performed, and you can only check whether the item has been checked under the remote menu.
It's easy, I want to add a status report for this, which only requires two lines of code.
But before that I want to hear from you.

Thanks

**Comment by tech10** [comment]
@cary-rowen If you know where the code needs to be added, and how to add it so that compatibility with NVDA 2021.1 and earlier can be maintained, I'm sure a PR would be reviewed, and hopefully, merged for this. Sounds like a good feature to me.
